run.py

```diff
@@ -30,6 +30,11 @@ def parse_args():
         type=str,
         help="Key to match the process name to stop the experiment",
     )
+    parser.add_argument(
```

Comment:
What't purpose of this option? Can you give you a example?

Reply:
For example: Legacy torch such as 1.13 only accepts --nproc_per_node instead of --nproc-per-node.